src/tests/roc_fec/target_openfec/test_writer_reader.cpp

```diff
@@ -797,5 +755,186 @@ TEST(writer_reader, delayed_packets) {
     }
 }
 
+TEST(writer_reader, fill_sn_sbn) {
```

Comment:
We have a test named "encode_source_id_and_seqnum" so I suggest to rename this one to "encode_sbn" and move it closer to that test.

Reply:
Done